src/signals/incident-management/containers/Dashboard/hooks/useDepartments.ts

```diff
@@ -12,6 +12,7 @@ import type { Department } from 'types/api/incident'
 import type { DepartmentDetails } from 'types/api/incident'
 import type { Category } from 'types/api/incident'
 
+import configuration from '../../../../../shared/services/configuration/configuration'
```

Comment:
Can be written as `shared/services/configuration/configuration`. Same for the utils. 

Actually double import with line 9